### DISCUSSION TITLE
Raspberry PI 4 - will only output on HDMI / Intermittently available

### DISCUSSION THREAD

**Discussion by lestat1977uk** [mikebrady/shairport-sync#1262] [Q&A] [answered]
Hi All, 

Really cool software but i could really use some help, been through all the guides and not getting anywhere.\

I've gotten the airplay to appear a few times but it always disappears and even when it does work it only works on HDMI, i'd really like to have it use 3.5 jack.

any help would be very much appricated

**Comment by iRayanKhan** [comment, marked as answer]
I've had it work using the 3.5mm jack, but with static feedback in the audio on a Pi 3B. I used a clean install of RaspberryPiOS. Make sure you have PuleAudio/Alsa. 

I have an HDMI DAC arriving later today to test HDMI, and static issue.

**Reply by mikebrady** [reply]
This is really an issue. I've copied it over to Issue 1300 and will delete this discussion item soon.
